tests/test_front_end_messages.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from app.frontend_messages import get_message
+from flask.ext.babel import gettext as _
+
+EXPECTED_MESSAGE = {
+                  'TITLE': _("Information"),
+                  'MSG': _("Unfortunately you can only complete one survey at a time."),
+                  'INSTRUCTIONS': _("Close this window to continue with your current survey."),
+                  }
+
+class TestFrontEndMessage(unittest.TestCase):
+    def test_front_end_messages(self):
```

Comment:
This test seems a bit redundant. i.e. Does the get_message lookup the only message we have coded it to return?